certificate-generation/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+#https://github.com/Magnitus-/DockerFiles/blob/master/certificate/example/ca-signed/docker-compose.yml
+version: "3"
+services:
+    ca-certificate-generator:
+        image: magnitus/certificate-generator:latest
+        environment:
+            COUNTRY: CA
+            STATE: Quebec
+            CITY: Montreal
+            ORGANIZATION: Any
+            DEPARTMENT: IT
+            EMAIL: email@cadomain.com
+            DOMAINS: dev.cadomain.com;test.cadomain.com
+            CERTIFICATE_DURATION: 1095
+            KEY_FILE: "ca.key"
+            CSR_FILE: "ca.csr"
+            CERTIFICATE_FILE: "ca.crt"
+            OUTPUT_CERTIFICATE_INFO: "true"
+        volumes:
+            - .:/opt/output
+    certificate-generator:
+        image: magnitus/certificate-generator:latest
```

Comment:
can you push image to chusj instead so we have control

Reply:
Its a private repo that I created during my own time for personal needs well before my employment here.

The source repo will always remain in my account, will point to my docker hub and will remain available under a MIT license.

I can do either of those things:
- Create a fork of my Dockerfiles repo for chusj and point to chusj there
- Create a script to tag and push the image to chusj